tidy3d/plugins/smatrix/component_modelers/terminal.py

```diff
@@ -263,13 +276,21 @@ def port_current(port: LumpedPort, sim_data: SimulationData) -> xr.DataArray:
             h1_field = h_field.isel({orth_component: orth_index - 1})
             h2_field = h_field.isel({orth_component: orth_index})
             h_field = h1_field - h2_field
+
+            # Find exact bounds of port taking into consideration the Yee grid
```

Comment:
a bit more explanation?

Reply:
I need to modify this computation anyways in #1542, and it should be much simpler.

Reply:
Is this PR closed in favor of #1542. It seems the same change happening there already?

Reply:
No I was planning on merging this PR soon, because it fixes some issue with coarse grids. Part of the issue was in this current computation. There is just a bit of overlap with  #1542. I think the code in #1542 will be much more clear with the usage of the path integrals.

Reply:
sounds good. feel free to merge.